dialogflow-cx/README.md

```diff
@@ -8,21 +24,21 @@ to create a webhook for a
 
 ### Install dependencies
 
-1. Install pyenv: https://github.com/pyenv/pyenv#installation
+1. Install pyenv: <https://github.com/pyenv/pyenv#installation>
 1. Use pyenv to install
     [the latest version of Python 3](https://www.python.org/downloads/) for
     example, to install Python version 3.10.1, run: `pyenv install 3.10.1`
 1. Create a Python virtual environment with the installed version of Python 3,
     for example, to create a Python 3.10.1 virtual environment called
     `dialogflow-webhook`, run: `pyenv virtualenv 3.10.1 dialogflow-webhook`
-1. Clone this repo and `cd` to the root of the repo
+1. Clone this repository and `cd` to the root of the repository
 1. Configure pyenv to use the virtual python environment we created earlier when
-    in this repo: `pyenv local dialogflow-webhook`
+    in this repository: `pyenv local dialogflow-webhook`
 1. Install the prerequisites: `pip install -r requirements.txt`
 
 ### Setup Google Cloud
 
-1. Install the Cloud SDK: https://cloud.google.com/sdk/docs/install
+1. Install the Cloud SDK: <https://cloud.google.com/sdk/docs/install>
 1. Run `gcloud init`, to
```

Comment:
I ran `gcloud auth application default login` here. It's not yet compatible with Python 3.10, so I had to use Python 3.9 in my virtual environment instead.

Reply:
Sorry, I should have clarified. I think the README should tell the user to create the virtual environment with Python 3.9. So lines `30-33`